build/run-e2e-tests-policy-framework-prow.sh

```diff
@@ -41,7 +41,9 @@ if [[ "${TARGET_BRANCH}" ]] && [[ "${TARGET_BRANCH}" != "main" ]]; then
   VERSION_TAG="${VERSION_TAG}-${PULL_BASE_REF#*-}"
 fi
 
+export KUBECONFIG=${HUB_KUBE}
```

Comment:
Where does `HUB_KUBE` come from?

Reply:
It's defined above this line.

Reply:
More specifically, `hub-<idx>.kc` and `managed-<idx>.kc` kubeconfigs are generated during the Prow workflow.